Tedu-106 - Added contacts section (#98)
mautic/user-documentation: docs/contacts/preference_center.rst

#################

> .. vale on

Reviewer: Vale off is not required here. Please remove.

.. vale off

> Manage Contact Preferences

Reviewer: ```suggestion
Manage Contact preferences
```

**************************

.. vale on

> When managing a Contact you can set the Contacts preference of communication. You can access the Contact's preference center when viewing a Contact's profile. From the dropdown menu click the preference menu. A new modal window should appear with a tab to set the preferred Channels and frequency of Contact, as well as the option to pause communication within a given period of time. 

Reviewer: ```suggestion
When managing a Contact you can set the Contact's preference of communication. You can access the Contact's Preference Center when viewing a Contact's profile. From the dropdown menu click the preference menu. A new modal window should appear with a tab to set the preferred Channels and frequency of Contact, as well as the option to pause communication within a given period of time. 
```

The second tab gives the option to add or remove the Contact from global Categories used in Emails or Categories. 

> The third tab allowS to add or remove the Contact from Segments it belongs to.

Reviewer: ```suggestion
The third tab allows you to add or remove the Contact from Segments they belong to.
```

.. vale off

Preferred Channels and frequency

> ********************************

Reviewer: H3 (===) rather than H2 as we should nest this in the above section.

.. vale on

.. image:: images/preferences.png
    :align: center
    :alt: Screenshot of Preference

|

In this window you can switch Channels of communication, set the frequency of the communication via each Channel enabled, and set one of the Channels as a preferred Channel.

> To set a Channel as a don't contact me through this Channel, un-tick the tick-box next to the Channel name in the first column.

Reviewer: ```suggestion
To prevent communications through a Channel, remove the check mark next to the Channel name in the first column. This sets a Do Not Contact (DNC entry) for only that channel.
```

> When a Channel is selected, these will be used to send marketing messages if there is a message set for any of the Channels selected. You can also set the frequency of the communication, as in this example the frequency is set to "Send me emails twice a day" but pause them during the "``1st`` of November 2022 to the ``30th`` of November 2022" . Email is also set as the preferred Channel, so if the same message is set for both email and ``sms``, it will only send the Email version of the message to the selected Contact.

Reviewer: ```suggestion
When a Channel is selected, these will be used to send Marketing Messages if there is a message set for any of the Channels selected. You can also set the frequency of the communication, as in this example the frequency is set to "Send me emails twice a day" but pause them during the "``1st`` of November 2022 to the ``30th`` of November 2022" . Email is also set as the preferred Channel, so if the same message is set for both Email and ``sms``, it will only send the Email version of the message to the selected Contact.
```

.. vale off

Contact Categories

> ******************

Reviewer: H3 (===) rather than H2

.. vale on

.. image:: images/categories.png
    :align: center
    :alt: Screenshot of Categories

|

> Use the Categories tab to add or remove a Contact from a global Category. Global Categories can be used in areas like Emails, text messages, Campaigns. In combination with the new Subscribed Categories Segment filter, Contacts can be given the choice to opt out of categorized communications.

Reviewer: ```suggestion
Use the Categories tab to add or remove a Contact from a global Category. Global Categories can be used in areas like Emails, Text Messages and Campaigns. In combination with the new Subscribed Categories Segment filter, Contacts can be given the choice to opt out of categorized communications.
```

.. vale off

Contact Segments

> ****************

Reviewer: H3 (===) rather than H2

.. vale on

.. image:: images/segments.png
    :align: center
    :alt: Screenshot of Segments

|

> Use the Segments tab to add or remove a Contact from a Segment. Segments are used as a source for Campaigns and Emails. Any Contact in a particular Segment will be part of a Campaign that has that Segment as the source. You can also use a standalone Email manually to a Segment. If a User has opted out of a Segment it will no longer receive Campaign actions or messages sent to that Segment.

Reviewer: ```suggestion
Use the Segments tab to add or remove a Contact from a Segment. Segments are used as a source for Campaigns and Emails. Any Contact in a particular Segment will be part of a Campaign that has that Segment as the source. You can also use a standalone Email to manually send an Email to a Segment. If a User has opted out of a Segment they will no longer receive Campaign actions or messages sent to that Segment.
```

.. vale off

> Contact's Unsubscribe Email Preferences

Reviewer: ```suggestion
Contact's unsubscribe Email preferences
```

> ***************************************

Reviewer: H3 (===) rather than H2

.. vale on

.. image:: images/email-unsubscribe-settings.png
    :align: center
    :alt: Screenshot of Email unsubscribe

|

> The Contact's preferences can be presented to the User in the unsubscribe page by selecting "Show Contact preference settings" in the Email configuration. You may also choose to hide or show different Segments of the User preferences. If any of these areas is set to no, it hides it from the contact's personal preferences page. The default unsubscribe message is shown if the preference setting option is set to no.

Reviewer: Might be worth linking this section to the part in the global configuration about the email preference centre options?

.. image:: images/unsubscribe.png
    :align: center
    :alt: Screenshot of Unsubscribe

|

Customize preference center

> ===========================

Reviewer: H2 (***) was correct here.

It's possible to customize the personal Preference Center/unsubscribe page, edit text labels, format, and apply Themes using the landing page builder.

.. vale off 

Creating a Preference Center Landing Page

> *****************************************

Reviewer: H3 (===) rather than H2 here.

.. vale on

When creating/editing a landing page, there is a toggle switch labeled Is *Preference Center*. If selected, the page will be marked as a preference center landing page.

When this page is configured as a preference center in a Mautic Email, recipients will be shown the page when clicking on the ``{unsubscribe_url}`` link. It also shows or hides the Preference Center slots in the builder.

.. image:: images/pref1.png
    :align: center
    :alt: Screenshot of Preference

|

Builder slots
*************

These slots in the builder are used to customize the page:

.. image:: images/pref2.png
    :align: center
    :alt: Screenshot of Preference

|

Preference tokens
******************

Optionally, you can use :doc:`tokens</configuration/variables>` to insert the different slots. Keep in mind that if you use :doc:`tokens</configuration/variables>`, you lose the ability to customize the labels and styles of the slots because it uses the default ones.

.. image:: images/pref3.png
    :align: center
    :alt: Screenshot of Preference

|

> See the :doc:`VARIABLES</configuration/variables>` page for a full list of tokens.

Reviewer: Let's link directly to the section for the preference centre tokens using the heading?

Author: I am kinda confused here. Please can you elaborate?

Reviewer: Instead of just dumping at the top of the page using https://mautic-documentation.readthedocs.io/en/latest/configuration/variables.html let's take them to where they need to be, eg: https://mautic-documentation.readthedocs.io/en/latest/configuration/variables.html#preference-center-landing-page-tokens using the anchor heading.

In addition, add a **Save preferences** button if you wish to save the preferences, otherwise the preferences can't be saved:

Save the page and the Preference Center landing page is ready.

.. vale off 

Landing Pages
*************

.. vale on

> Now in the landing pages list, the icon with the little cog indicates that the page is a Preference Center one.

Reviewer: ```suggestion
Now in the landing pages list, the icon with the cog icon indicates that the page is a Preference Center.
```
